data/virt_autotest/setup_dns_service.txt

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+
+source /usr/share/qa/qa_test_virtualization/shared/standalone
+wait_script="30"
+bridgeif="br123"
+bridgeip="192.168.123.1";
+dns_domain_forward="testvirt.net";
+dns_domain_reverse="123.168.192";
+dhcpd_lease_file="/var/lib/dhcp/db/dhcpd.leases"
+setup_log_file="/var/log/virt_dns_setup.log"
+
+if [ -e "$setup_log_file" ];then
+	rm -f "$setup_log_file"
+fi
+
+zyppercmd="zypper --non-interactive in lsb-release bridge-utils bind bind-chrootenv bind-utils"
+echo -e "$zyppercmd will be executed\n" | tee -a ${setup_log_file}
+$zyppercmd
+
+#Add repo to upgrade bash to v4.3, otherwise legacy system may not support associative array
+reposarch=`arch`
+bashrepourl="http://download.suse.de/ibs/SUSE/Updates/SLE-SERVER/12-SP4/${reposarch}/update/"
+zyppercmd="zypper --non-interactive --no-gpg-checks ar ${bashrepourl} bashrepo"
+echo -e "${zyppercmd} will be executed\n" | tee -a ${setup_log_file}
+${zyppercmd}
+zyppercmd="zypper --non-interactive --gpg-auto-import-keys ref"
+echo -e "${zyppercmd} will be executed\n" | tee -a ${setup_log_file}
+${zyppercmd}
+zyppercmd="zypper --non-interactive update bash"
+echo -e "${zyppercmd} will be executed\n" | tee -a ${setup_log_file}
+${zyppercmd}
+zyppercmd="zypper --non-interactive rr bashrepo"
+echo -e "${zyppercmd} will be executed\n" | tee -a ${setup_log_file}
+${zyppercmd}
```

Comment:
Suggest to follow naming rules for variables , for separate words, use '_' or big capital to start with, eg zypperCmd or zypper_cmd.

Reply:
Done